libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -49,6 +49,13 @@ public class MicrosoftAppCredentials : ServiceClientCredentials
         /// </summary>
         private readonly Lazy<AdalAuthenticator> authenticator;
 
+        /// <summary>
+        /// Creates a new instance of the <see cref="MicrosoftAppCredentials"/> class.
+        /// </summary>
+        /// <param name="appId">The Microsoft app ID.</param>
+        /// <param name="password">The Microsoft app password.</param>
+        public MicrosoftAppCredentials(string appId, string password) : this(appId, password, null) { }
```

Comment:
Same comment here. The other ctor needs to remove the "= null" that's making it a default.